src/TagsEditor.cpp

```diff
@@ -243,7 +243,7 @@ TagsEditorDialog::TagsEditorDialog(wxWindow * parent,
    Fit();
    Center();
    wxSize sz = GetSize();
-   SetSizeHints(sz.x, std::min(sz.y, 600));
+   SetSizeHints(sz.x, std::min(sz.y, 600),-1,wxDisplay().GetGeometry().GetHeight()-100);
```

Comment:
OK for its effects

Stylistically, I'd prefer the name `wxDefaultCoord` to the magic number -1, spaces after each comma and around binray `-`, and a line break to keep it in 80 columns.